src/expressions/include/antares/expressions/visitors/InvalidNode.h

```diff
@@ -22,11 +22,11 @@
 #pragma once
 #include <stdexcept>
 
-namespace Antares::Solver::Visitors
+namespace Antares::Expressions::Visitors
 {
 class InvalidNode: public std::invalid_argument
 {
 public:
     explicit InvalidNode(const std::string& node_name = "");
```

Comment:
Dangerous to have a default argument here (`node_name = ""`).
This means we allow : 
```C++
throw InvalidNode();
```
without specifying any node name, and in this case, if exception is caught, associated error message will be cryptic, as we don'y know which node is concerned.